src/lib/y2packager/dialogs/addon_selector.rb

```diff
@@ -107,9 +124,10 @@ def dialog_content
           # TRANSLATORS: Product selection label (above a multi-selection box)
           Left(Heading(_("Available Extensions and Modules"))),
 
-          VWeight(75, MinHeight(12,
+          VWeight(60, MinHeight(8,
```

Comment:
can you also add screenshot of ncurses version? as screen looks a bit busy for me.

Reply:
Ncurses screenshot added to the PR description. :arrow_up: